pyproject.toml

```diff
@@ -27,6 +27,8 @@ classifiers = [ # TODO(cookiecutter): Remove and add more if appropriate
 requires-python = ">= 3.11, < 4"
 dependencies = [ # TODO(cookiecutter): Remove and add more if appropriate
   "typing-extensions >= 4.5.0, < 5",
+  "astroid >= 2.15.4, < 2.17.0",
```

Comment:
The TODO above those lines should probably be removed

Reply:
* #13

Reply:
I have created a new PR for this